oneview/resource_fc_network.go

```diff
@@ -131,6 +152,38 @@ func resourceFCNetworkCreate(d *schema.ResourceData, meta interface{}) error {
 		d.SetId("")
 		return fcNetError
 	}
+	// updates connection_template
+	if rawVal, ok := d.GetOk("bandwidth"); ok {
+		bandwidthVal := rawVal.([]interface{})
+		for _, bandwidth := range bandwidthVal {
+			rawBandwidth := bandwidth.(map[string]interface{})
+			// get ethernet network by name
+			fcNet, er := config.ovClient.GetFCNetworkByName(d.Get("name").(string))
+			if er != nil {
+				log.Printf("unable to get fc network for connection_template_uri: %s", er)
```

Comment:
just put the msg as "unable to get FC network: %s", er

Reply:
I have specifically mentioned that because this call has been made in the read method also. So it is better to be specific about an error.

Reply:
ok got it